Phonebook.Frontend/src/app/shared/components/navigation/navigation.component.html

```diff
@@ -29,7 +29,9 @@
         </mat-list-item>
         <mat-list-item>
           <button mat-menu-item (click)="navigateToOwnProfile(); sidenav.close()">
-            <mat-icon>perm_identity</mat-icon>
+            <mat-icon *ngIf="currentUser == null">perm_identity</mat-icon>
+            <app-profile-picture *ngIf="currentUser != null" [imageSize]="50" class="pb-self-centered circle" [user]="currentUser">

```

Comment:
```suggestion
            <app-profile-picture *ngIf="currentUser != null" [imageSize]="50" class="mat-icon pb-self-centered circle" [user]="currentUser">
```

Reply:
this way you don't need your css `#pb-nav-mobile app-profile-picture`

Reply:
but app-profile-picture is clearly not a mat-icon. It does not seem right to add inappropriate classes just to avoid some css.

Reply:
It effectivly is, it works just as the other icons on the buttons? Just that it is bigger than the other icons.